src/main/java/common/Carrier.java

```diff
@@ -10,7 +10,7 @@
  */
 public abstract class Carrier {
     protected final Integer carrierId;
-    protected final List<RfPort> rfPorts;
+    protected List<RfPort> rfPorts = null;
```

Comment:
This doesn't need to be explicitly initialized to null here.

Reply:
When we were unit testing, it errored out without the explicit initialization, but I just tried again and it's working fine without so I'll take this out. Must have been something weird when we were still testing our change.